### ISSUE TITLE
is it support optional chaining ?

### ISSUE THREAD

**Issue by yeongjet** [babel/eslint-plugin-babel#178]
.eslintrc:
```
{
    "parser": "babel-eslint",
    "plugins": ["babel"],
    "rules": {
        "babel/camelcase": 1
    }
}
```
package.json
```
"babel-eslint": "^8.2.6",
"eslint": "^4.19.1",
"eslint-plugin-babel": "^5.3.0"
```
why I still got error

**Comment by ljharb** [comment]
You’d want eslint 5 regardless.

**Comment by yeongjet** [comment]
upgrape eslint to 5.16 still not work

**Comment by nicolo-ribaudo** [comment]
What error?

**Comment by theblang** [comment]
@nicolo-ribaudo Weird, I'm actually not seeing `babel/no-unused-expressions` work for optional chaining either. I've installed `eslint-plugin-babel 5.3.0` and have `eslint 6.6.0` installed. I've also added `babel` to `plugins` as the instructions said: `"plugins": ["babel", "prettier"]`. Side note, we do use the `eslint-config-airbnb`, but I don't *think* they make use of this plugin, and our `eslint` should override those rules anyway.

I've tried just installing / configuring and not specifying a rule, as well as setting `babel/no-unused-expressions` to both `1` and `0`. In all cases I see `no-unused-expressions` lint errors for `someArray?.push('another new item');`

**Update:** I figured this out as I was typing the comment, so I'll still post just for future viewers of this issue. I had to turn off the original rule: `"no-unused-expressions": ["off"], // defer to babel plugin`. Then specify whatever value you want in the babel version; for us it was: `"babel/no-unused-expressions": ["error"]`. Now I'm seeing the behavior we want, where the first line here errors but the second does not:

```
  someArray && someArray.push('new item');
  someArray?.push('another new item');
```

cc @yeongjet -- Maybe a similar situation in your case?
